### PR TITLE
Add Common/Known issues to ReadME

### PR DESCRIPTION
**Pull request by OlafSzmidt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,6 +113,9 @@ Even with these basic mechanics, there is quite a lot of complexity in creating
 * [This blog post](http://mkelsey.com/2013/04/30/how-i-setup-virtualenv-and-virtualenvwrapper-on-my-mac/) may also be
  useful.
 
+## Common/Known Issues
+* If you get an error saying `no module named google.auth` after trying to run `./run.py` or `./run.py -k`, rerun the command again and the dependancy should be detected. This is a logged issue and we are working to solve it.
```
Comment (CelineBoudier) [inline README.md]: dependency

Comment (CelineBoudier) [reply README.md]: looged issue: provide a reference?

Comment (OlafSzmidt) [reply README.md]: Thought it was, made it now: #449

File: README.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ Even with these basic mechanics, there is quite a lot of complexity in creating
 
 ## Running Locally
 * Clone the repo
-* Make and activate a virtualenv (We recommend [virtualenvwrapper](http://virtualenvwrapper.readthedocs.org/en/latest/index.html)) - if you have a **Mac see the following section**.
+* Make and activate a virtualenv (We recommend [virtualenvwrapper](http://virtualenvwrapper.readthedocs.org/en/latest/index.html)) - if you have a **Mac see the section at the bottom**.
```
Comment (mrniket) [inline README.md]: could link this

**Pull request merged**